### DISCUSSION TITLE
Startup error: `Package 'irobot_create_gazebo_bringup' not found`

### DISCUSSION THREAD

**Discussion by bdawkinsroj** [iRobotEducation/create3_docs#591] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

latest humble release

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

Package 'irobot_create_gazebo_bringup' not found: "package 'irobot_create_gazebo_bringup' not found, searching: ['/opt/ros/humble']" , is the error I keep running into when trying to launch the create3 sim in Gazebo. I have followed all instructions prior.

**Comment by alsora** [comment, marked as answer]
Hi @bdawkinsroj your error indicate that you didn't install the create 3 simulator correctly.

Please try the following, and if it still doesn't work send the full console output you got while running the commands.
```sh
sudo apt-get update
sudo apt-get install -y ros-humble-irobot-create-gazebo-sim ros-humble-irobot-create-nodes ros-humble-irobot-create-gazebo-bringup
```

**Reply by bdawkinsroj** [reply]
It works now! Thank you!
